lmnet/executor/train.py

```diff
@@ -18,8 +18,9 @@
 
 import click
 import tensorflow as tf
-from tensorflow.core.util.event_pb2 import SessionLog
 
+from multiprocessing import cpu_count
```

Comment:
Please change import order following pep8 👍 
https://www.python.org/dev/peps/pep-0008/#id23

Reply:
I've fixed it.